src/ORM/Connect/Database.php

```diff
@@ -386,6 +388,7 @@ protected function escapeColumnKeys($fieldValues)
      */
     public function manipulate($manipulation)
     {
+        $this->extend('onBeforeManipulate', $manipulation);
```

Comment:
Small question. Should we also update out documentation with information that `Database` subClasses can have extensions applied to them and they can use `onBeforeManipulate` method?

Reply:
It's such a niche extension hook that I don't think it's even worth mentioning in the 6.0.0 changelog.